packages/schematics/angular/library-entrypoint/index_spec.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { SchematicTestRunner, UnitTestTree } from '@angular-devkit/schematics/testing';
+import { Schema as LibraryOptions } from '../library/schema';
+import { Schema as WorkspaceOptions } from '../workspace/schema';
+import { Schema as GenerateLibrarySchema } from './schema';
+import { parse as parseJson } from 'jsonc-parser';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function getJsonFileContent(tree: UnitTestTree, path: string): any {
+  return parseJson(tree.readContent(path).toString());
+}
+
+describe('Secondary Entrypoint Schematic', () => {
+  const schematicRunner = new SchematicTestRunner(
+    '@schematics/ng_packagr',
+    require.resolve('../collection.json'),
+  );
+  const defaultOptions: GenerateLibrarySchema = {
+    name: 'foo-secondary',
+    project: 'foo',
+  };
+
+  const workspaceOptions: WorkspaceOptions = {
+    name: 'workspace',
+    newProjectRoot: 'projects',
+
+    version: '6.0.0',
+  };
+  const libaryOptions: LibraryOptions = {
+    name: 'foo',
+    entryFile: 'my-index',
+    standalone: true,
+    skipPackageJson: false,
+    skipTsConfig: false,
+    skipInstall: false,
+  };
+
+  let workspaceTree: UnitTestTree;
+  beforeEach(async () => {
+    workspaceTree = await schematicRunner.runSchematic('workspace', workspaceOptions);
+  });
+
+  it('should create correct files', async () => {
+    workspaceTree = await schematicRunner.runSchematic('library', libaryOptions, workspaceTree);
```

Comment:
Run `workspaceTree = await schematicRunner.runSchematic('library', libaryOptions, workspaceTree);` in `beforeEach`